src/distance.cpp

```diff
@@ -0,0 +1,81 @@
+#include "attitude-utils.hpp"
+#include "style.hpp"
+#include "camera.hpp"
+#include "distance.hpp"
+#include <math.h>
+
```

Comment:
Your includes should be in a specific order, and seperated by the hierarchy of categories:
- Any C/C++ standard libraries (with are surrounded by <>)
- Any External libraries that you're using (doesn't apply here, but would be something like Python's numpy)
- Any libraries that you've defined outside of this folder (also doesn't apply here)
- Any libraries that you've defined inside this folder that are not this header file (e.g. style.hpp)
- The header file that corresponds to this one
```
#include <math.h>

#include "attitude-utils.hpp"
#include "style.hpp"
#include "camera.hpp"

#include "distance.hpp"
```